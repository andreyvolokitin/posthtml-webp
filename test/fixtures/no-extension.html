<!doctype html>
<html>
    <body>
        <img src="photo.png">
        <amp-img alt="photo" width="550" height="368" layout="responsive" src="photo.png"></amp-img>
    </body>
</html>
