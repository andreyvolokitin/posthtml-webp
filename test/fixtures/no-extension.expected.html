<!doctype html>
<html>
    <body>
        <picture><source type="image/webp" srcset="photo.png.webp"><img src="photo.png"></picture>
        <amp-img alt="photo" width="550" height="368" layout="responsive" src="photo.png.webp"><amp-img alt="photo" width="550" height="368" layout="responsive" src="photo.png" fallback=""></amp-img></amp-img>
    </body>
</html>
